build.gradle

```diff
@@ -46,19 +45,21 @@ configurations {
 			cacheChangingModulesFor 0, "seconds"
 		}
 	}
+	intermediary
 }
 
 dependencies {
 	enigmaRuntime "net.fabricmc:stitch:0.2.1.61"
-	enigmaRuntime "cuchaz:enigma:0.14.2.134:all"
+	enigmaRuntime "cuchaz:enigma:0.14.2.134"
+	intermediary "net.fabricmc:intermediary:$minecraft_version"
+	intermediary "net.fabricmc:intermediary:$minecraft_version:v2"
```

Comment:
I have a feeling this will break in a future gradle update as it is similar to the issue with the lwjgl natives. For now leave it be if you dont have a better idea.

Reply:
I actually think this might not be needed anymore? let me check

Reply:
We can keep it the  way it was before where it downloaded from Github, is that better?